standard/sections/clause_7_normative_text.adoc

```diff
@@ -202,18 +203,18 @@ include::../recommendations/core/REC_extent_geospatial_precision.adoc[]
 
 ===== Additional Geospatial Extents
 
-A WCMP record is able to communicate additional extents (for example, for non-geographic data, or multiple
+A WCMP record can communicate additional extents (for example, for non-geographic data or multiple
 temporal extents) via the `+properties.extent+` property.  The `+properties.extent.spatial.bbox+` property
-provides the ability to list 1..n minimum bounding geometries for a given dataset, where:
+provides the ability to list 1 to many minimum bounding geometries for a given dataset, where:
 
-* minx: westernmost coordinate of the limit of the dataset extent, expressed in longitude decimal degrees as a
-signed number between -180 and 180, less than or equal to maxx
-* miny: southernmost coordinate of the limit of the dataset extent, expressed in latitude decimal degrees as a
-signed number between -90 and 90, less than or equal to maxy
-* maxx: easternmost coordinate of the limit of the dataset extent, expressed in longitude decimal degrees as a
-signed number between -180 and 180, greater than or equal to minx
-* maxy: northernmost coordinate of the limit of the dataset extent, expressed in latitude decimal degrees as a
-signed number between -90 and 90, greater than or equal to miny
+* *minx* is the westernmost coordinate of the limit of the dataset extent, expressed in longitude decimal degrees as a
```

Comment:
Remove trailing periods from this bulleted list

Reply:
keeping periods in bulleted lists.